pkg/api/graph/test/hpa-missing-cpu-target.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: autoscaling/v1
+apiVersion: autoscaling/v2alpha1
```

Comment:
v1 does defaulting during conversion to internal, v2alpha1 (and the former extensions/v1beta1) does not. The test won't see the missing cpu target anymore.